Documentation/Getting-Started-with-Modules-Part-1.markdown

```diff
@@ -0,0 +1,417 @@
+
+## Course structure
+Throughout the course we will alternate between discussing topics and implementing them. The discussion may contain example code or other example scenarios. 
+
+So that there is no confusion for you as to what you should be doing, when it comes to implementing these lessons into the module it will be explained step-by-step via numbered lists.
+
+Later on in the course, as the topics become more advanced, we may go several sections of discussion before wrapping up the lessons into changes to the codebase.
+
+You will also occasionally come across **Bonus Exercise** sections. These are completely optional. You can skip them, complete them at the time, or come back after completing the course to complete them. They are suggested when there is an extra feature you could implement using the skills you have just learned.
+
+## Getting started
+Now that you have completed all of the setup tasks you will have a fresh copy of Orchard configured and ready to go. 
+
+The rest of this part of the course will walk you through the process required to scaffold an empty module and then build a simple `Widget` inside of it.
+
+## Command line scaffolding with Orchard.exe
+You should now be looking at Visual Studio. Down the side, in your Solution Explorer window you will see many files and folders. 
+
+The first step to take is to collapse all of the projects down. Its a long list and we need to be able to see an overview of the solution so we can start working with it. You don't need to collapse these individually by hand however:
+
+  1. If your Solution Explorer window is not visible click `View`, `Solution Explorer`. 
+  
+  1. Click the `collapse all` icon in the top of the solution explorer which looks like this:
+     
+    ![](../Attachments/getting-started-with-modules-part-1/collapse-all.png)
+
+If you open up your `Modules` folder you will see a long list of the modules which come packaged with Orchard:
+
+![](../Attachments/getting-started-with-modules-part-1/modules-list.png)
+
+There is a utility that is packaged with each copy of Orchard which will let us add our own module into this list, `orchard.exe`. This is a command line utility which will scaffold up a new empty module and add it to the main solution. There are also other commands you can use with this utility. 
+
+To scaffold a new module:
+
+  1. Press the `Save All` button (or press `Ctrl-Shift-S`). Its a good practice to always save before using the command line utility. Many of its commands will make changes to your solution and if you have unsaved changes you will get merge conflicts.
+  
+  1. In the Solution Explorer, scroll down to the `Orchard.Web` project.
+  
+  1. Right click on `Orchard.Web` project and choose `Open Folder in File Explorer`:
+     
+     ![](../Attachments/getting-started-with-modules-part-1/open-orchard-web.png)
+  
+  1. Locate `orchard.exe` in the list and double click it to open.
```

Comment:
You can also specify here 
>"Locate 'orchard.exe' **under the bin folder** and double click it to open". 

(It's not right in the 'Orchard.Web' folder so it will save some seconds of searching) what do you think?